src/Nethermind/Ethereum.VM.Test/BitwiseLogicOperationTests.cs

```diff
@@ -33,7 +33,7 @@ public void Test(GeneralStateTest test)
         
         public static IEnumerable<GeneralStateTest> LoadTests() 
         { 
-            var loader = new TestsSourceLoader(new LoadGeneralStateTestsStrategy(), "vmArithmeticTests");
+            var loader = new TestsSourceLoader(new LoadGeneralStateTestsStrategy(), "vmBitwiseLogicOperation");
```

Comment:
name changed?

Reply:
No. Github is reading it strange way. We have vmArithmeticTests as well, but this one is treated by github as file changed from previous ArithmeticTests.
Genaraly all VM tests are now removed from Blockchain.Test and Blockchain.Block.Test and are only in VM.Test, in changed (fixed) format. It is why there are some strange changes showed by github